src/BenchmarkDotNet/Disassemblers/ClrMdV2Disassembler.cs

```diff
@@ -8,16 +8,35 @@
 using System.Linq;
 using System.Text.RegularExpressions;
 using BenchmarkDotNet.Portability;
+using BenchmarkDotNet.Helpers;
 
 namespace BenchmarkDotNet.Disassemblers
 {
     // This Disassembler uses ClrMd v2x. Please keep it in sync with ClrMdV1Disassembler (if possible).
     internal abstract class ClrMdV2Disassembler
     {
-        // Translating an address to a method can cause AV and a process crash (https://github.com/dotnet/BenchmarkDotNet/issues/2070).
-        // It was fixed in https://github.com/dotnet/runtime/pull/79846,
-        // and most likely will be backported to 7.0.2 very soon (https://github.com/dotnet/runtime/pull/79862).
-        protected static readonly bool IsVulnerableToAvInDac = !RuntimeInformation.IsWindows() && Environment.Version < new Version(7, 0, 2);
+        private static readonly ulong MinValidAddress = GetMinValidAddress();
+
+        private static ulong GetMinValidAddress()
+        {
+            // https://github.com/dotnet/BenchmarkDotNet/pull/2413#issuecomment-1688100117
+            if (RuntimeInformation.IsWindows())
+                return 65536;
+            if (RuntimeInformation.IsMacOS())
+                return RuntimeInformation.GetCurrentPlatform() switch
+                {
+                    Environments.Platform.X86 or Environments.Platform.X64 => 4096,
+                    Environments.Platform.Arm64 => 0x100000000,
+                    _ => throw new NotSupportedException($"{RuntimeInformation.GetCurrentPlatform()} is not supported")
+                };
+            if (RuntimeInformation.IsLinux())
+            {
+                string? minAddrResult = ProcessHelper.RunAndReadOutput("/bin/bash", "sudo cat /proc/sys/vm/mmap_min_addr");
```

Comment:
Why not File.ReadAllText("/proc/sys/vm/mmap_min_addr")?  (be sure to catch exceptions)

Either way, I don't think that'll work on all flavors of linux.  You might need a coarser check,

Reply:
Or maybe it's not protected after all. I tried without sudo and it worked.

Reply:
This check would not work for dump debugging in case you are debugging it on a different distro from the one the dump was taken on and they differed in the null zone size. I am not sure if it is worth trying to be precise here. Runtime itself uses the target page size on Unix and 64kB on Windows to determine whether an invalid access is a null reference or an access violation, so I guess the same thing would be sufficient here.

Reply:
Ah, right, I've forgotten that this is for benchmarkdotnet usage only. So you are right, this is for live processing only. 
You can get the page size using `Environment.SystemPageSize`

Reply:
@janvorli Thanks. Do you know if the value for Android is the same as Linux, and iOS the same as MacOS?

Reply:
I don't know, but I don't think we support disassembling on any of these. AFAIK the disassembler supports coreclr and coreclr doesn't support them.